apps/pie-storybook/package.json

```diff
@@ -13,15 +13,16 @@
     "@justeattakeaway/pie-button": "workspace:*"
   },
   "devDependencies": {
-    "@storybook/addon-essentials": "7.0.0-beta.40",
-    "@storybook/addon-links": "7.0.0-beta.40",
-    "@storybook/blocks": "7.0.0-beta.40",
-    "@storybook/web-components": "7.0.0-beta.40",
-    "@storybook/web-components-vite": "7.0.0-beta.40",
+    "@storybook/addon-essentials": "7.0.7",
```

Comment:
I remember having to downgrade to beta.40 a couple of months ago because of a build error, let me check if that still happens before we merge this.

Reply:
funnily enough, I couldn't seem to hook into the toolbar/globals API without upgrading!

Reply:
@xander-marjoram I've unresolved this because I think I've managed to get it to a more useful state which would make sense to merge. If you could help me test that windows build issue that would be awesome, please :)

Reply:
That said, we do now run the build against windows in the OS matrix. Perhaps a successful build there indicates there's no longer an issue?

Reply:
I'm pretty confident that if CI is passing, you won't have issues locally. Feel free to try locally though, we all know how troublesome your machine is @xander-marjoram 😉